flocker/node/agents/test/test_blockdevice.py

```diff
@@ -2845,6 +2845,19 @@ def test_mountpoint_error(self):
         failure = self.failureResultOf(mount_result, OSError)
         self.assertEqual(ENOTDIR, failure.value.errno)
 
+    def test_mountpoint_permissions(self):
+        """
+        The mountpoint is world-writeable (since containers can run as any
+        user), and its parent is only accessible as current user (for
+        security).
+        """
+        mountroot = mountroot_for_test(self)
+        mountpoint = mountroot.child(b"mount-test")
+        self._run_success_test(mountpoint)
+        self.assertEqual((mountroot.getPermissions().shorthand(),
+                          mountpoint.getPermissions().shorthand()),
+                         ('rwx------', 'rwxrwxrwx'))
```

Comment:
I also meant to mention "Sticky Bit":
 * http://en.wikipedia.org/wiki/Sticky_bit

But not sure if that would make things any more secure or whether it would add complication.